src/widgets/mentors-register/ui/mentors-register.module.scss

```diff
@@ -1,6 +1,15 @@
 .mentoring-register {
   display: flex;
   flex-direction: column;
+  max-width: 680px;
+
+  &-section {
```

Comment:
Please, rename classes - your `mentoring-register` is inside `mentoring-register-section`